app/src/models/destination.tsx

```diff
@@ -16,6 +17,7 @@ const defaultValues: IDestinationStore = {
     lng: undefined,
     price: undefined,
     cityName: undefined,
+    personalPriorityIdx: undefined,
```

Comment:
No "undefined" types

Reply:
how do I define default values then? using _null_ instead? I would love to get rid of it, but that Record thing from immutable requires default object

Reply:
I don't'want to come up with any default coordinates, because there's a chance of missing a check at some point and they will be displayd on a map